### DISCUSSION TITLE
Can I add external property for git push command

### DISCUSSION THREAD

**Discussion by inchara-1998** [microsoft/vscode-discussions#745] [Extension Development QnA] [answered]
We can add issues/JIRA number which we are working to push command from command prompt.
Please refer below screenshot
![image](https://github.com/microsoft/vscode-discussions/assets/118963451/41cea891-5062-4488-83e6-838dacc00582)

Same thing i want to do from vs code using git extension once we open any git repository i am activating extension i am building and trying to show the issues/JIRA which user is working. but once they select that i should add selected issue in push command.

Can i know how to do is it possible.

please refer below image i am loading issues in workitem tab i need to pass that issue number as mentioned in error log.

![image](https://github.com/microsoft/vscode-discussions/assets/118963451/9c54dafd-c7ba-47d5-be5f-4f79d7c09409)

**Comment by inchara-1998** [comment]
Hi any update on this??

**Comment by isidorn** [comment]
This discussion is a place for extension authors. ANd it does not look like you are developing a VS Code extension.
I suggest to ask this question on StackOVerflow. And I will go ahead and close this. Thank you

**Comment by inchara-1998** [comment]
I am developing a extension from our extension only we are loading work item tabs
So for next step we need to map that issues while pushing 
So i am asking is it possible

**Reply by r3m0t** [reply, marked as answer]
Look at the git extension API

https://github.com/microsoft/vscode/blob/main/extensions/github/src/typings/git.d.ts#L299

Or maybe a `pre-push` hook in Git itself
